src/lib.rs

```diff
@@ -56,7 +56,7 @@ mod util;
 
 /// The result type used to describe all fallible operations of the scryfall
 /// crate.
-pub type Result<T> = std::result::Result<T, error::Error>;
+pub type Result<T> = std::result::Result<T, Box<error::Error>>;
```

Comment:
`crate::Result` is much smaller that way, since one of the variants of `Error` is quite large. An alternative approach would be to define `Error` as `pub struct Error(Box<ErrorImpl>)`.

Reply:
The serde error impl is already boxed (I poked around their source code) so boxing here creates two indirections.

Maybe we could box just the large variant?

Reply:
That sounds like a bit of a micro-optimization. If the crate if working properly the serde error shouldn't even come up so performance isn't critical.

Anyway, done.